e2e_test.go

```diff
@@ -0,0 +1,79 @@
+//go:build e2e
+
+/*
+Copyright 2022 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package celvet_test
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/DangerOnTheRanger/celvet"
+)
+
+func TestE2E(t *testing.T) {
+	testFilePaths, err := filepath.Glob("testdata/e2e/*_test.yaml")
```

Comment:
Alternative if you don't want to have to repeat the directory path so much in the code:

```
testDir := fmt.Sprintf("testdata/e2e")
testFiles, err := ioutil.ReadDir(testDir)
```

Reply:
Since `ReadDir` is going to return `[]DirEntry` and `DirEntry.Name()` returns the filename (not the entire path), wouldn't the full path still have to be repeated with `filepath.Join(...)` or something along those lines?

Reply:
The regex becomes simpler when you are only processing file names and then actually open a file just prepend testDir? `filepath.Join` seems fine for that.

Reply:
Marking as resolved now that `os.ReadDIr`/`filepath.Join` are used.